.buildkite/bk.integration-fips.pipeline.yml

```diff
@@ -81,6 +81,7 @@ steps:
               - "true"
             groups:
               - fleet # currently there is only a single test in the fleet group, add more tests once they have been defined
+              - ech-deployment # TODO: remove once tested in PR
```

Comment:
@ycombinator please remember to remove this when you are done with your testing 🙂

Reply:
Thanks for catching; removed in b5196b3917.